scintellometry/io/arochime.py

```diff
@@ -327,39 +338,34 @@ def seek_record_read(self, offset, size):
             raise ValueError("size and offset must be an integer number of records")
 
         raw = self.fh_raw.seek_record_read(offset, size)
-
-        if self.npol == 2:
```

Comment:
Am a bit confused that this can be removed. I assume it still works on 2-pol data of both the old and new format?